src/Agent.Worker/Build/TfsVCCommandManager.cs

```diff
@@ -236,6 +236,20 @@ private string FormatArguments(FormatFlags formatFlags, params string[] args)
             return string.Join(" ", formattedArgs);
         }
 
+        private string GetEndpointData(ServiceEndpoint endpoint, string name)
+        {
+            var trace = HostContext.GetTrace(nameof(SourceProvider));
```

Comment:
Agree.. using Trace.
I should have been a little more diligent while copying code.